kubernetes/aks/system/oneuptime/argo-app.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: oneuptime
+  namespace: argocd
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  destination:
+    namespace: oneuptime
+    server: https://kubernetes.default.svc
+  sources:
+    - chart: oneuptime
+      helm:
+        releaseName: oneuptime
+        valueFiles:
+          - $values/kubernetes/aks/system/monitoring/helm/values.yaml
```

Comment:
`/kubernetes/aks/system/oneuptime/helm/values.yaml` (but maybe we could put oneuptime in monitoring? )

Reply:
typoooo

Reply:
Fixed. I thought about it at first but I prefer to keep it separated for visibility.